api/core/model_runtime/model_providers/wenxin/llm/ernie-4.0-8k.yaml

```diff
@@ -1,12 +1,12 @@
-model: ernie-bot
+model: ernie-4.0-8k
```

Comment:
a model name should not be changed once it has been used for a period of time, otherwise it will lead the configuration of some apps to be invalid.